### DISCUSSION TITLE
I am having issues understanding the SaaS option for traefik in coolify

### DISCUSSION THREAD

**Discussion by firstaxel** [coollabsio/coolify#2171] [Q&A]
traefik.enable=true
traefik.http.routers.<unique_router_name>.rule=HostRegexp(`{subdomain:[a-zA-Z0-9-]+}.coolify.io`)
traefik.http.routers.<unique_router_name>.entryPoints=https
traefik.http.routers.<unique_router_name>.middlewares=gzip
traefik.http.routers.<unique_router_name>.service=<unique_service_name>
traefik.http.services.<unique_service_name>.loadbalancer.server.port=80
traefik.http.routers.<unique_router_name>.tls=true
traefik.http.routers.<unique_router_name>.tls.certresolver=letsencrypt
traefik.http.routers.<unique_router_name>.rule=HostRegexp(`{subdomain:[a-zA-Z0-9-]+}.coolify.io`)
traefik.http.routers.<unique_router_name>.entryPoints=http
traefik.http.routers.<unique_router_name>.middlewares=redirect-to-https

I understand that is the needed settings but I don't understand how to set it up and also they used coolify.io here for both proxy setup and for the SaaS configuration

I was thinking how can I use different domain for setting up the SaaS Traefik Configuration.

**Comment by soufian212** [comment]
You fix it ? 
Im facing the same issue

**Comment by djsisson** [comment]
if youre using traefik v3, this syntax no longer works you need to use :
```yaml
rule=HostRegexp(`^.+\.coolify\.io$`)
```

**Reply by echoeyecodes** [reply]
OMG thank you so much!!!!!!!!!
